cert_manager/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "display_name": "cert-manager",
+  "maintainer": "ara.pulido@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "cert_manager",
+  "metric_prefix": "cert_manager.",
+  "metric_to_check": "cert_manager.prometheus.health",
+  "creates_events": false,
+  "short_description": "Track all your cert-manager metrics with Datadog",
+  "guid": "c9bdaf11-fe15-4892-ae30-47c5124144e5",
+  "support": "contrib",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-cert_manager Integration",
+  "categories": [
+    ""
```

Comment:
We should probably have at least one category. Does [`cloud`](https://docs.datadoghq.com/integrations/#cat-cloud) sound appropriate, or perhaps [`configuration & deployment'](https://docs.datadoghq.com/integrations/#cat-configuration-deployment)?